README.md

```diff
@@ -54,12 +54,17 @@ xc dev    # opens the app in dev mode
 
 # Internationalization / Translations
 
-We need help translating our user interface into different languages. The translation related source code are all in `./svelte/src/libs/translations/*`.
+We’d love your help in translating the gui into different languages.
+The translation related source code are all in `./svelte/src/libs/translations/*`.
```

Comment:
If you want to enforce style, `markdownlint` can help here. [MD013](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md) (line length) can help with this one.

(Note: I love linters for keeping things consistent.)

Reply:
fine with me

Reply:
#719